util/util.go

```diff
@@ -100,80 +90,6 @@ func Ordinalize(n int) string {
 	return fmt.Sprintf("%d%s", n, m[an%10])
 }
 
-// ParseMajorMinorPatch returns a semver.Version from the string provided
-// by looking only at major.minor.patch and stripping everything else out.
-// Deprecated: Please use the function in util/version
-func ParseMajorMinorPatch(v string) (semver.Version, error) {
-	return version.ParseMajorMinorPatchTolerant(v)
-}
-
-// ModifyImageRepository takes an imageName (e.g., repository/image:tag), and returns an image name with updated repository
-// Deprecated: Please use the functions in util/container
-func ModifyImageRepository(imageName, repositoryName string) (string, error) {
```

Comment:
there's some overlap with #4005 here, I don't mind rebasing on top of it once it merges.

This is ready for review in the meantime though.

/hold

Reply:
rebased on top of #4078

/hold cancel